index.js

```diff
@@ -1,7 +1,28 @@
 const flat = require('flat');
 const safeJson = require('json-stringify-safe');
-
-exports.log = function(options, tags, logStatement) {
+const chalk = require('chalk');
+const defaults = {
```

Comment:
not sure I want app color to decide if all colors off or on. Lets add a color: true|false (default to false).  I think you can use chalk for everything and disable it in the constructor so the code can be basically same for enabled or disabled